src/user/contributing.rst

```diff
@@ -49,4 +49,20 @@ We are glad to know that you would like to contribute to the ``conda-forge`` doc
 5. Submit a `pull request
    <https://help.github.com/articles/about-pull-requests/>`_ to the main repository proposing your changes.
 
+
+   Writing guidelines:
```

Comment:
I would delete these spaces in front of "Writing guidelines" here. They aren't needed.